src/single-movie-component/single-movie-component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+function SingleMovieView(props) {
+  const movie = props.movie;
+  return (
+    <section className="single-movie-dashboard">
```

Comment:
We can look into the dependency that allow us to embed youtube videos.

Reply:
Instead of embedding, you can just use an HTML video element! Neat